inject.js

```diff
@@ -1346,17 +1346,18 @@ function addTechnicalNames() {
 
                 var fieldType = jQuery(this).closest('[type]').attr('type') || jQuery(this).text().toLowerCase();
                 var btn = '';
```

Comment:
Heh, I actually didn't notice this wasn't being used anywhere, and maybe you already had thought of doing something like this.